core/autoscaling.go

```diff
@@ -8,6 +8,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/autoscaling"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"math"
+)
+
+const (
+	OnDemandPercentageLong  = "on_demand_percentage"
+	OnDemandNumberLong      = "on_demand_number"
```

Comment:
That's fine, the combined mechanism was actually first proposed when we had a much smaller limit on tags, but in the meantime it was increased and now it is more than enough. I don't think it will become a problem anytime soon.